src/components/virtual-scroll/virtual-util.ts

```diff
@@ -442,58 +440,54 @@ export function writeToNodes(plt: Platform, nodes: VirtualNode[], cells: Virtual
   }
 }
 
-
 /**
  * NO DOM
  */
 export function adjustRendered(cells: VirtualCell[], data: VirtualData) {
-  // figure out which cells should be rendered
-  let cell: VirtualCell;
-  let lastRow = -1;
-  let cellsRenderHeight = 0;
-  let maxRenderHeight = (data.renderHeight - data.itmHeight);
-  let totalCells = cells.length;
-  let viewableRenderedPadding = (data.itmHeight < 90 ? VIEWABLE_RENDERED_PADDING : 0);
+
+  const maxRenderHeight = (data.renderHeight - data.itmHeight);
+  const totalCells = cells.length;
+  const viewableRenderedPadding = (data.itmHeight < 90 ? VIEWABLE_RENDERED_PADDING : 0);
 
   if (data.scrollDiff > 0) {
     // scrolling down
     data.topCell = Math.max(data.topViewCell - viewableRenderedPadding, 0);
-    data.bottomCell = Math.min(data.topCell + 2, totalCells - 1);
+    data.bottomCell = data.topCell;
 
-    for (var i = data.topCell; i < totalCells; i++) {
-      cell = cells[i];
-      if (cell.row !== lastRow) {
```

Comment:
The deletion of this, causes the virtual-scroll not to work correctly when using multi-column. Now the PR is already merged and ionic 4 is coming, so this is just a comment to let other people know how they can fix it.